modules/tensor_mechanics/doc/content/source/materials/cohesive_zone_model/CZMRealVectorCartesianComponent.md

```diff
@@ -0,0 +1,26 @@
+# CZMRealVectorCartesianComponent
+
+!syntax description /Materials/CZMRealVectorCartesianComponent
+
+## Description
+
+This is a Material model used to extract components of a real vector  defined on a cohesive zone in a Cartesian coordinate system. This can be used regardless of the coordinate
+system used by the model.
+
+This Material model is set up by [CohesiveZoneMaster](CohesiveZoneMaster/index.md) automatically
+when traction or displacement-jump components are requested in the generate_output parameter, but can also be set up directly by the user.  
+
+The `CZMRealVectorCartesianComponent` takes as arguments the values of the `index` to save into a MaterialProperty.  [eq:real_vecto_indices] shows the component values for each vector component.
```

Comment:
Do we really need this? We can use `MaterialRealVectorValueAux` to fetch the material properties.

Reply:
this is also how the TM action works: first, it generates all scalar material properties, then it adds MaterialRealAux. I simply followed it, and I asked myself the same question about `RankTwoAux`. Let me know